http/basic/src/main/java/org/wildfly/security/http/basic/BasicAuthenticationMechanism.java

```diff
@@ -60,6 +62,7 @@
 
     private final boolean includeCharset;
     private final String configuredRealm;
+    private final Logger log = Logger.getLogger(BasicAuthenticationMechanism.class);
```

Comment:
Thanks for the PR, @vijay-boddupalli!

Just a small comment. Instead of adding this logger, you should be able to make use of the `httpBasic` logger that's used elsewhere in this file, i.e., you should be able to use `httpBasic.trace...`